java/src/main/java/com/google/oak/transparent_release/RekorLogEntry.java

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 //
 
-package com.google.oak.rekor;
+package com.google.oak.transparent_release;
```

Comment:
Can we come up with a better package name? How about `transparency`?

Reply:
Done